math/vecops/inc/ROOT/RVec.hxx

```diff
@@ -1143,6 +1143,23 @@ RVec<T> Take(const RVec<T> &v, const int n)
    return r;
 }
 
+/// Return a copy of the container without the elements at the specified indices.
+template <typename T>
+RVec<T> Drop(const RVec<T> &v, RVec<typename RVec<T>::size_type> idxs)
+{
+   RVec<T> r;
+   r.reserve(v.size() - idxs.size());
+   std::sort(idxs.begin(), idxs.end());
+   auto discardIt = idxs.begin();
+   for (auto i = 0u; i < v.size(); ++i) {
```

Comment:
Shoudn't `i` be of type `typename RVec<T>::size_type`?

Reply:
indeed, thanks!